### PR TITLE
a bunch of dependency updates

### PR DESCRIPTION
**Pull request by jensens**
lets see what happens on jenkins ... :)

### CODE REVIEW

**Comment by jensens** [comment]
the zope.i18nmessageid test problem is addressed here plone/plone.supermodel#18 and here plone/plone.schemaeditor#45

File: versions.cfg [hunk 1]
```diff
@@ -40,14 +43,15 @@ Products.Sessions = 3.0
 
 #############
 # Build Tools
-
+
```
Comment (gforcada) [inline versions.cfg]: @jensens would you mind removing this?

File: versions.cfg [hunk 2]
```diff
@@ -25,9 +25,12 @@ ZopeUndo = 4.1
 coverage = 4.2
 zdaemon = 4.1.0
 transaction = 1.4.4
-Sphinx = 1.3.6
+Sphinx = 1.5.1
+unittest2 = 1.1.0
```
Comment (gforcada) [inline versions.cfg]: @jensens why is unittest2 reintroduced? I've been trying to remove it for a while

File: versions.cfg [hunk 3]
```diff
@@ -40,14 +43,15 @@ Products.Sessions = 3.0
...
-
+ 
 # Basics
 # !! keep in sync with requirements.txt !!
 setuptools = 26.1.1
 zc.buildout = 2.5.3
 
 # recipes and extensions
 collective.recipe.omelette = 0.16
+collective.recipe.sphinxbuilder = 1.0
```
Comment (gforcada) [inline versions.cfg:54]: @jensens do you remember who needs it?

**Review by gforcada — COMMENTED** [review]
Nice work, I added some questions though :-)

File: versions.cfg (hunk 3 above)
Comment (jensens) [reply versions.cfg:54]: It is used in the main ``buildout.cfg`` of ``buildout.coredev``

File: versions.cfg (hunk 2 above)
Comment (jensens) [reply versions.cfg]: it is used by robotsuite and http://dist.plone.org/versions/zopetoolkit-1-0-8-ztk-versions.cfg pins a very old 0.5.1

File: versions.cfg (hunk 1 above)
Comment (jensens) [reply versions.cfg]: sure

**Comment by jensens** [comment]
- [x] merge plone/Products.Archetypes#72
- [x] wait for release of ``robotsuite`` and get rid of unittest2 (and its dependencies)

File: versions.cfg (hunk 2 above)
Comment (jensens) [reply versions.cfg]: see also collective/robotsuite#12 - then we can unpin. wait for release

**Comment by mauritsvanrees** [comment]
For the record, the `html5lib` version increase gave errors when using `zest.releaser`. I fixed that in ec2cb40f6905ab049ce38dd896f18693e91b14a0. I did update the version as much as possible, effectively from 1.0b7 to 1.0b8.

**Comment by jensens** [comment]
html5lib and its versioning crap is a big PITA. Sorry for the inconvenience.

**Pull request merged**